.github/workflows/ci-workflow.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         shell: bash
   test_windows:
     name: Testing on Windows
-    runs-on: windows-2016
+    runs-on: windows-2019
```

Comment:
May this be the reason for the test failure on windows ?

Reply:
I changed this because it was also failing on `windows-2016`. I hoped that `windows-2019` would fix it, but alas...